src/test/scala/com/codacy/api/FileHelperTest.scala

```diff
@@ -0,0 +1,43 @@
+package com.codacy.api
+
+import com.codacy.api.helpers.FileHelper
+import org.scalatest.{FlatSpec, Matchers}
+
+class FileHelperTest extends FlatSpec with Matchers {
+
+  "FileHelper" should "withTokenAndCommit" in {
+
+    val result: Either[String, Boolean] = FileHelper.withTokenAndCommit(projectToken = Some("PROJECT-TOKEN-DUMMY-VALUE")) {
+      case (projectToken, commitUUID) =>
+
+        projectToken shouldNot be('empty)
+        commitUUID shouldNot be('empty)
+
+        Right(true)
+    }
+
+    result should be('right)
+
+    result match {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/Codacy/codacy-api-scala/file/1247389679/issues/source?bid=2272948&fileBranchId=2935685#l21)